lib/modules/platform/gitlab/index.ts

```diff
@@ -806,6 +808,16 @@ export async function updatePr({
     body.target_branch = targetBranch;
   }
 
+  // Add new labels to the PR
+  if (addLabels) {
+    body.add_labels = addLabels;
+  }
+
+  // Remove old labels from the PR
+  if (removeLabels) {
+    body.remove_labels = removeLabels;
+  }
+
```

Comment:
It feels a bit off to have comments to describe the code here. Would creating two new functions like `addNewLabelsToPR` and `removeOldLabelsFromPR` be better?

Reply:
I think it is pretty clear what's happening a new function will only add extra lines..not much beenfit. I will remove the comment though, they add nothing extra  :)